core/src/main/python/compare_model.py

```diff
@@ -232,6 +232,12 @@ def debug(format_string, *arguments):
     else:
         _logger.finest(format_string, arguments)
 
+def _checkModelExtension(file):
```

Comment:
python function names should be Snake case, not camel case.

Reply:
fixed